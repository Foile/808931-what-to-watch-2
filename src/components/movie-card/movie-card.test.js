import React from 'react';
import renderer from "react-test-renderer";
import MovieCard from './movie-card';

it(`MovieCard correctly renders after relaunch`, () => {
  const movie = {id: 1, name: `Macbeth`, previewImage: ``, videoLink: ``};
  const tree = renderer
  .create(<MovieCard
    movie = {movie}
    onHeaderClick = {()=>jest.fn()}
    onMouseEnter={()=>jest.fn()} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});

