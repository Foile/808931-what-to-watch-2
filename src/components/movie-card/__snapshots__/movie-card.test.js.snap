// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard correctly renders after relaunch 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    className="small-movie-card__link"
    href="/films/1"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        className=""
        controls={false}
        height="180"
        muted={true}
        poster=""
        width="280"
      >
        <source
          src=""
          type="video/mp4"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      Macbeth
    </h3>
  </a>
</article>
`;
